### DISCUSSION TITLE
tbl-scap

### DISCUSSION THREAD

**Discussion by DavidJesse21** [quarto-dev/quarto-cli#4565] [Feature Requests]
Hi,

for plots/figures that produce pdf output, Quarto provides the cell options "fig-cap" and "fig-scap".
The former is the main caption that appears beneath the plot and the latter (if used) is the figure caption that is displayed in the list of figures ([see here](https://quarto.org/docs/reference/cells/cells-knitr.html#figures)).

I was wondering if there exists a similar feature for tables ([I don't think so](https://quarto.org/docs/reference/cells/cells-knitr.html#tables)) or if that is planned.
It should technically not differ much, right?

Cheers
David

**Comment by emfeltham** [comment]
Hi, I would just like to reiterate that this would be a very useful feature for academic writing, where one wouldn't want a long caption to appear in the List of Tables. One alternative that works now would be to use `tbl-subcap`, and have the longer content there, and a shorter one in the `tbl-cap`. Although, this seems to change the table format in general, making it hard (or not possible?) to have tables that spans multiple pages because of a shift from from `longtable` to `tabular`.

**Reply by rich-iannone** [reply]
The equivalent feature for tables will be worked on for v1.4. This will be after the work on crossref interactions (which will unblock the work on sub captions).

**Reply by ghost** [reply]
@rich-iannone Thanks for the info.  Do we have an issue that captures this now we can link in?

**Reply by rich-iannone** [reply]
@allenmanning There is an issue epic (https://github.com/quarto-dev/quarto-cli/issues/4944) and something akin to that here: https://github.com/quarto-dev/quarto-cli/issues/4171. Many more as well, like this one: https://github.com/quarto-dev/quarto-cli/issues/4168. I feel this needs to be organized under the issue epic at https://github.com/quarto-dev/quarto-cli/issues/4944 since there are a lot of these.

**Reply by cscheid** [reply]
Have we confirmed that `tbl-scap` doesn't work?

**Reply by emfeltham** [reply]
It hasn't worked for me, at least (as it does for figures).

I am stuck between using the `subcap` approach or having the full caption appear in the List of Tables.

**Reply by cscheid** [reply]
Thanks for the confirmation and patience. I'm going to be working on crossrefs throughout all of our 1.4 release, and I hope to fix this by then.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/4944
